### PR TITLE
framework/bluetooth: Include config.h header file

### PR DESCRIPTION
**Pull request by Wootak**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ac3d9b3f32bb340a514e207e06815d8c8e35f9b5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ac3d9b3f32bb340a514e207e06815d8c8e35f9b5] - Code Rule Check (C++) OK.

File: framework/src/bluetooth/adaptation/bt-adatation-adapter.c [hunk 1]
```diff
@@ -15,6 +15,7 @@
  *
  */
 
+#include <tinyara/config.h>
```
Comment (sunghan-chang) [inline framework/src/bluetooth/adaptation/bt-adatation-adapter.c]: There is no CONFIG_XX definition. Could you let me know the reason of this commit?

Comment (Wootak) [reply framework/src/bluetooth/adaptation/bt-adatation-adapter.c]: This is template code. So, CONFIG_XX definition can be added in later.
and, for consistency, I added this line all of bluetooth files.

**Pull request merged**